src/layout/header/Menu.tsx

```diff
@@ -1,75 +1,93 @@
-import '../../styles/layout/menu.scss';
-
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { useHistory } from 'react-router';
 
-import { Button, Menu as MUIMenu, MenuItem } from '@material-ui/core';
+import { Button, Menu as MuiMenu, MenuItem } from '@material-ui/core';
 import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
 import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+import styled from 'styled-components';
 
 interface MenuProps {
   handleLogout: () => void;
   menuButtonLabel: string;
 }
 
+const StyledMenu = styled.div`
+  grid-area: menu;
+`;
+
+const StyledMuiMenu = styled(MuiMenu)`
+  .MuiMenu-list {
+    width: 15rem;
+    border: 3px solid ${({ theme }) => theme.palette.box.main};
+
+    @media only screen and (max-width: 600px) {
+      width: 10rem;
+    }
+  }
+`;
+
+const StyledMenuButton = styled(Button)`
+  width: 15rem;
+
+  @media only screen and (max-width: 600px) {
+    width: 10rem;
+  }
+`;
+
+const StyledMenuItem = styled(MenuItem)`
+  border-bottom: 1px solid ${({ theme }) => theme.palette.box.main};
+`;
+
 const Menu: React.FC<MenuProps> = ({ menuButtonLabel, handleLogout }) => {
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
-  const [menuOpen, setMenuOpen] = React.useState(false);
   const { t } = useTranslation();
   const history = useHistory();
 
   const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
     setAnchorEl(event.currentTarget);
-    setMenuOpen(!menuOpen);
   };
 
   const handleClose = () => {
     setAnchorEl(null);
-    setMenuOpen(false);
   };
 
   return (
-    <div className="menu">
-      <Button
+    <StyledMenu>
+      <StyledMenuButton
         color="primary"
         variant="contained"
         aria-controls="menu"
         aria-haspopup="true"
         onClick={handleClick}
         data-testid="menu">
         {menuButtonLabel}
-        {menuOpen ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
-      </Button>
-      <MUIMenu
+        {anchorEl ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+      </StyledMenuButton>
+      <StyledMuiMenu
         anchorEl={anchorEl}
-        elevation={0}
         getContentAnchorEl={null}
         keepMounted
-        open={!!anchorEl}
+        open={Boolean(anchorEl)}
```

Comment:
kan vi bruke !! istedenfor Boolean? jeg tenker siden vi bruker &&, ||, === osv, så er det mer naturlig å bruke !!

Reply:
k. bingo-bango for meg